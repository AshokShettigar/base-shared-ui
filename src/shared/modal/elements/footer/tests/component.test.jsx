import React from 'react';
import { shallow } from 'enzyme';

import ModalFooterElement from '../component';

const defaultProps = {
    submitLabel: '',
    dismissLabel: '',
    onSubmit: jest.fn(),
    onDismiss: jest.fn(),
    isDisabled: false,
    isLoading: false
};

it('should render component', () => {
    const wrapper = shallow(<ModalFooterElement {...defaultProps} />);

    expect(wrapper).toMatchSnapshot();
});
