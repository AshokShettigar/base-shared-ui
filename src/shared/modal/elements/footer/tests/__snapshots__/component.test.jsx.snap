// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<div
  className="root"
  data-component="ModalFooterElement"
>
  <WithRender
    className="previous"
    hidden={true}
    kind="secondary"
    rendered={true}
    size="medium"
  />
  <WithRender
    className="submit"
    disabled={false}
    hidden={false}
    kind="primary"
    loading={false}
    onClick={[Function]}
    rendered={true}
    size="medium"
    type="submit"
  />
  <WithRender
    className="dismiss"
    hidden={false}
    kind="secondary"
    onClick={[Function]}
    rendered={true}
    size="medium"
  />
</div>
`;
